Packages/com.unity.inputsystem/InputSystem/Actions/InputActionParameters.cs

```diff
@@ -0,0 +1,805 @@
+        /// from bindings.
+        ///
+        /// <example>
+        /// <code>
+        /// // Create an action map with two actions.
+        /// var map = new InputActionMap();
+        /// var action1 = map.AddAction("action1");
+        /// var action2 = map.AddAction("action2");
+        ///
+        /// // Add a binding to each action to which  a "ClampProcessor" is applied.
+        /// // This processor has two parameters:
+        /// // - "min" (float)
+        /// // - "max" (float)
+        /// action1.AddBinding("&gt;Gamepad&gt;/rightTrigger", processors: "clamp(min=0.2,max=0.8)");
+        /// action2.AddBinding("&gt;Gamepad&gt;/leftTrigger", processors: "clamp(min=0.2,max=0.8)");
+        ///
+        /// // Apply parameter overrides to set the values differently.
+        /// // This will apply the setting to *both* the bindings on action1 *and* action2.
+        /// map.ApplyParameterOverride("min", 0.3f);
+        /// map.ApplyParameterOverride("max", 0.9f);
+        /// </code>
+        /// </example>
+        ///
+        /// An override can optionally be directed at a specific type of object.
+        ///
+        /// <example>
+        /// <code>
+        /// map.ApplyParameterOverride("clamp:min", 0.3f);
+        /// map.ApplyParameterOverride("clamp:max", 0.9f);
+        /// </code>
+        /// </example>
+        ///
+        /// By default, the parameter override will apply to all bindings in the map. To limit the override
+        /// to specific bindings, you can supply a <paramref name="bindingMask"/>.
+        ///
+        /// <example>
+        /// <code>
+        /// // Apply a parameter override only to action1.
+        /// map.ApplyBindingOverride("clamp:min", 0.25f, new InputBinding { action = action1.name });
+        ///
+        /// // Apply a parameter override only to a specific binding path.
+        /// map.ApplyBindingOverride("clamp:min", 0.4f, new InputBinding { path = "&lt;Gamepad&gt;/leftTrigger" });
+        /// </code>
+        /// </example>
+        ///
+        /// If there are multiple overrides that could be applied to a specific parameter, the most specific override
+        /// is chosen. For example, if both an override for "duration" and an override for "duration" specifically on bindings
+        /// to the "look" action exists, then the latter is used for all bindings to the "look" action and the former is used
+        /// for all other bindings. For parameter overrides of equal specificity, the behavior is undecided.
+        ///
+        /// Note that parameter overrides stay in place on the map. Like binding overrides, however, they are not
+        /// automatically persisted and thus need to be reapplied when actions are loaded from assets. This will, however, be applied
+        /// automatically to bindings added to the action in the future as well as whenever bindings are resolved to controls.
```

Comment:
I guess this answers my question stated in the beginning of this PR.... that overrides are applied to processors/interactions added after the override was set.